src/text/vector_text.js

```diff
@@ -0,0 +1,18 @@
+const vectorText = require('./vectorText')
+
+/** Construct an array of character segments from a ascii string whose characters code is between 31 and 127,
+* if one character is not supported it is replaced by a question mark.
+* @param {Float} x - x offset
+* @param {Float} y - y offset
+* @param {String} text - ascii string
+* @returns {Array} characters segments [[[x, y], ...], ...]
+* @deprecated >= v2
+*
+* @example
+* let textSegments = vector_text(0, -20, 'OpenJSCAD')
+*/
+function vector_text (x, y, text) {
```

Comment:
Ironically the naming of the functions is the only thing that I think should be changed:
- we dropped all 'snake_case' in favor of cameCase for V2 (see docs and all our changes, so the 'outward facing' api should be using that for coherence
- which brings me to a second point : why are we not just using vectorText(options, text) etc ? 
   * that is coherent for V2
   * no need to wrap anything
   * api is more rich (options object)

All the above  is also applicable for vectorChar

Reply:
One thing I am not 100% sure about is whether vectorChar should be exposed or not : I do not see a difference between a string of a single caracter and a special case for just one char(vectorChar), so for simplicity I think only ### **vectorString** should be exposed

Reply:
@kaosat-dev thanks for the review.

Good. We seem to be in agreement now. The old style functions were dropped. The new style functions were retained. So, only vectorChar and vectorText remain.

You are correct about vectorChar and vectorText being equivalent. Not sure if vectorChar should be dropped as well. For now, we can keep both.

If you agree then this pull request is ready to merge.

Reply:
super, thanks !
ok let's keep it for now, but I am pretty much 100% we can/should/will remove vectorChar in the end :)